modules/test/playwright/tests/export-import-web/main/instance.import.spec.ts

```diff
@@ -104,6 +105,107 @@ test('can export and import custom object entries at instance level', async ({
 	);
 });
 
+test('can import account restricted entry when account does and does not exist in enviroment', async ({
+	apiHelpers,
+	companyExportImportPage,
+}) => {
+	const account = await apiHelpers.headlessAdminUser.postAccount();
+
+	apiHelpers.data.push({
+		id: account.id,
+		type: 'account',
+	});
+
+	const objectDefinition =
+		await apiHelpers.objectAdmin.postRandomObjectDefinition({
+			status: {code: 0},
+		});
+
+	const objectRelationshipAPIClient = await apiHelpers.buildRestClient(
+		ObjectRelationshipAPI
+	);
+
+	const {body: objectRelationship} =
+		await objectRelationshipAPIClient.postObjectDefinitionByExternalReferenceCodeObjectRelationship(
+			'L_ACCOUNT',
+			{
+				label: {
+					en_US: 'objectRelationshipLabel' + getRandomInt(),
+				},
+				name: 'objectRelationshipName' + Math.floor(Math.random() * 99),
+				objectDefinitionExternalReferenceCode1: 'L_ACCOUNT',
+				objectDefinitionExternalReferenceCode2:
+					objectDefinition.externalReferenceCode,
+				type: 'oneToMany',
+			}
+		);
+
+	apiHelpers.data.push({
+		id: objectRelationship.id,
+		type: 'objectRelationship',
+	});
+
+	const objectDefinitionAPIClient =
+		await apiHelpers.buildRestClient(ObjectDefinitionAPI);
+
+	await objectDefinitionAPIClient.patchObjectDefinition(objectDefinition.id, {
+		accountEntryRestricted: true,
+		accountEntryRestrictedObjectFieldName: `r_${objectRelationship.name}_accountEntryId`,
+	});
+
+	const applicationName = 'c/' + objectDefinition.name.toLowerCase() + 's';
+	const accountEntryERC = `r_${objectRelationship.name}_accountEntryERC`;
+	const accountEntryId = `r_${objectRelationship.name}_accountEntryId`;
+
+	const objectEntry = await apiHelpers.objectEntry.postObjectEntry(
+		{
+			[accountEntryERC]: account.externalReferenceCode.toString(),
+			[accountEntryId]: account.id.toString(),
+		},
+		applicationName
+	);
+
+	const exportFilePath = await companyExportImportPage.export(
+		`${objectDefinition.name} 1 Items`
+	);
+
+	await test.step('assert entry is imported with account relationship properties when it exists', async () => {
+		await apiHelpers.delete(
+			`${apiHelpers.baseUrl}${applicationName}/${objectEntry.id}`
+		);
+
+		await companyExportImportPage.import(exportFilePath);
+
+		const importedObjectEntry = await apiHelpers.get(
+			`${apiHelpers.baseUrl}${applicationName}/by-external-reference-code/${objectEntry.externalReferenceCode}`
+		);
+
+		expect(importedObjectEntry).toMatchObject({
+			[accountEntryERC]: account.externalReferenceCode,
+			[accountEntryId]: account.id,
+		});
+	});
+
+	await test.step('assert entry is imported wiht account relationship properties when it does not exist', async () => {
+		await apiHelpers.delete(
+			`${apiHelpers.baseUrl}${applicationName}/${objectEntry.id}`
+		);
+
+		await apiHelpers.headlessAdminUser.deleteAccount(account.id);
```

Comment:
Please also make sure the entry and account have been deleted. This is how we simulate different environments. Therefore, I think it's important to ensure the items have been deleted.